docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -96,21 +105,13 @@ Parameter Explanation:
 
 Because no invocations of the indicated method were found, the method is unnecessary and can be deleted.
 
-### UnusedMethodsRule Limitations
+### UnusedMethodRule Limitations
 
-- UnusedMethodsRule works on private or protected constructors and private instance methods.
+- UnusedMethodRule works on private or protected constructors and private instance methods.
 - These methods and constructors aren’t supported, are excluded from analysis, and may produce false negatives:
     - public or global constructors
     - protected, public, or global instance methods
-    - static methods of any visibility aren't currently supported
-
-#### See Also
```

Comment:
I've moved this back to ApexCrudFlsViolationRule since the information isn't relevant to UnunsedMethodRule.